### ISSUE TITLE
And an automated test to make sure the issues page is loading we can use a mock API

### ISSUE THREAD

**Issue by DonnieBLT** [OWASP/BLT-Flutter#355]

**Comment by Dante291** [comment]
@DonnieBLT Could you please explain this, do I have to write unit test to check issues load using mock API?

**Comment by DonnieBLT** [comment]
Yes, this would be part of the test suite that runs and flutter. I think we have some automated tests already, but we want to make sure that the issue page always loads. I'm not sure this test may already be done.

**Comment by Dante291** [comment]
> Yes, this would be part of the test suite that runs and flutter. I think we have some automated tests already, but we want to make sure that the issue page always loads. I'm not sure this test may already be done.

@DonnieBLT I could not find 'Tests' folder in the directory so this mean basically no tests are written for BLT-Flutter app, Testing is important for big apps like BLT as adding new features could break some part of the app. 

I am gonna start writing tests for this and will later make new issues to add more tests of important feature in BLT.

**Comment by DonnieBLT** [comment]
Sounds good thank you
